operator/charts/paladin-operator/values.yaml

```diff
@@ -93,13 +93,22 @@ paladin:
     repository: kaleidoinc/paladin
     tag: release
     pullPolicy: Always
+  labels:
+    app: paladin
+  
 besu:
   image:
     repository: hyperledger/besu
     tag: latest
     pullPolicy: Always
-postgres:
+  labels:
+    app: besu
+postgres: # the postgres container runs as a sidecar to the paladin container
   image:
     repository: postgres
     tag: latest
-    pullPolicy: Always
\ No newline at end of file
+    pullPolicy: Always
+
+# Install CRDs as part of the chart installation
+# When this is set to false, the CRDs are expected to be installed separately
+installCRDs: false
```

Comment:
📦

Reply:
Well there is still the option of having a single installation for both the CRDs and the oprtator